mut/redirects/redirect_main.py

```diff
@@ -89,219 +89,201 @@ def write_to_file(rules: List[RuleDefinition], output_path: str) -> None:
             f.write('\n')
 
 
+def parse_line(line, rc, line_num, version_regex, url_regex, dict_regex):
+    # strip \n from line
+    line = line.strip()
+
+    # allow for comments:
+    if (len(line) > 0 and line[0] != '#'):
```

Comment:
I'd probably restructure this condition:

```
if not line or line.startswith('#'):
    continue
...
```

Reply:
Fixed. Also moved this check to the parse_source_file() function to shave one more indentation.